### ISSUE TITLE
Service down?

### ISSUE THREAD

**Issue by iliapolo** [zeke/semantic-pull-requests#95]
Hello,

Ever since yesterday (May 17th), all my PR's are stuck waiting for the `Semantic Pull Request` status to be reported. 

I've tried opening a new PR today and the same thing happens. 

See https://github.com/aws/aws-cdk/pulls
See https://github.com/aws/aws-cdk/pull/8048

Is it possible the service is down? How can i get to the bottom of this? I can't seem to find any incidents page for your service. 

Thanks

**Comment by abatilo** [comment]
Could we get a little visibility to what's happening, please?

**Comment by vinayakkulkarni** [comment]
seems the service is up 🎉

**Comment by bodinsamuel** [comment]
Is there anything we can do to help / not depend on service that has no monitoring ? ☺️

**Comment by Nadock** [comment]
During the downtime I switched some repos I work on to use the GitHub Action mentioned in #90:
https://github.com/amannn/action-semantic-pull-request

So far so good, but I have only been using it for about a day.

**Comment by abatilo** [comment]
@felipecassiors or @zeke 

Can we get some kind of messaging for what happened?

**Comment by felipecassiors** [comment]
It seems to be up again, just tested here.

We tried to upgrade `conventional-commit-types` to 3.0.0 in order to drop the `improvement` type support. The first time it failed (https://github.com/zeke/semantic-pull-requests/pull/94) and then @zeke reverted the commit (https://github.com/zeke/semantic-pull-requests/pull/96).  We then tried again (https://github.com/zeke/semantic-pull-requests/pull/98).

Now the bot is up and running, and with the `conventional-commit-types` upgrade.

For further details (and logs) we need @zeke inputs.

**Comment by zeke** [comment]
Hey folks 👋 

TLDR:

- Sorry about that.
- Will try to stabilize more in the future.
- Consider migrating to a [GitHub Actions workflow](https://github.com/zeke/semantic-pull-requests/issues/90) for your own safety.

My apologies for the recent downtime. It was caused by a [merged pull request](https://github.com/zeke/semantic-pull-requests/pull/94/files) with some breaking changes that were not caught by the existing test suite.

I am pleased that this service is valuable to people and excited to see that so many developers are embracing semantic workflows for releasing software. On the flip side, this is a side project of mine, and the growing adoption of this service has pulled me away from my regular work.

The app was initially designed for use within the @electron org with a specific use case in mind: **Does this pull request have enough semantic information to trigger a semantic-release when squashed?** When this was the app's only behavior, things were more stable. But as it has grown in popularity, members of the community have come forth with slightly different use cases, and they've contributed changes that have made the application logic much more difficult to understand and maintain. I've made an effort to welcome these contributions and accommodate people's needs, but in hindsight maybe I should have been a bit more discerning about the contributions I've accepted.

Going forward, I will make an effort to be more careful and scrutinous in evaluating new pull requests, bearing in mind that breaking the service could affect many users.

I am also investigating GitHub Actions powered alternatives to this GitHub App. The distinct advantage of Actions in this case is that each user (you) has autonomous control over the configuration of their workflow, and are not beholden to some unknown third party (i.e. me) who may accidentally break things when the service goes down. There's an unfolding discussion about Actions-based alternatives here: https://github.com/zeke/semantic-pull-requests/issues/90

Thanks for reading, and keep up the good work. ✨ 🤖 ✨

**Comment by zeke** [comment]
To anyone affected by the outage mentioned in this issue, you may be happy to know that there's a (no longer buggy) GitHub Action that might suit your needs better: https://github.com/zeke/semantic-pull-requests/pull/111 -- switching could mean one less service dependency for you and your app! 🍰

**Comment by abatilo** [comment]
@zeke Is the service down again, by chance?

**Comment by zeke** [comment]
Not that I know of, @abatilo 

It was working this morning:

<img width="518" alt="Screen Shot 2020-10-07 at 3 07 12 PM" src="https://user-images.githubusercontent.com/2289/95393091-d53fec00-08ae-11eb-8254-fe02ec6ba453.png">

**Comment by abatilo** [comment]
Hmm. I have a build that's been stuck and was just updated an hour ago. Would the easiest way to re-test be just to push another commit? Is there some way I can hit some kind of endpoint with my org/repo information or something?

**Comment by zeke** [comment]
Changing the PR title should trigger a new check I think...
